### ISSUE TITLE
MSH HL7 Enhancements

### ISSUE THREAD

**Issue by MauriceReeves-usds** [CDCgov/prime-data-hub#453]
Per feedback from multiple states, there are several things that need to be address with the MSH segment in our HL7 messages:

- MSH-3 should be 20 characters or less
- MSH-4 should correspond to each specific facility and should be 20 characters or less
- MSH-5 should come from the organization. Currently we define it in the schema, but we should make it dynamic
- MSH-6 should come from the organization. Currently we define it in the schema, but we should make it dynamic
- MSH-7 should be populated - This is the file creation date
- MSH-11 should be populated - This signifies if the message is D/Debugging, T/Training, or P/Production

cc: @jlusds @RickHawesUSDS @mikepif

**Comment by MauriceReeves-usds** [comment]
Checked some changes into #404 that addresses the following issues:
- MSH-5 - I added a cli parameter that lets me pass in the receiving organization. This is a stop gap. I would like it to come from the organization or organization service, but for now this will work.
- MSH-6 - I added a cli parameter that lets me pass in the receiving facility. This is a stop gap. I would like it to come from the organization or organization service, but for now this will work.
- MSH-7 - The COVID-19 schema defines `file_created_date` that we don't currently request from SimpleReport, so I have set it to be the date and time the file is serialized to HL7. 
- MSH-11 - Currently coded to be 'D,' since we're in the process of onboarding, but we can set up different pipelines per a suggestion from @jimduff-usds and let that pass in this value to Hub
